### PR TITLE
chore(functions): address comments on CAL samples

### PR DESCRIPTION
**Pull request by ace-n**

### CODE REVIEW

File: functions/v2/index.js [hunk 1]
```diff
@@ -68,8 +69,14 @@ exports.helloAuditLog = cloudevent => {
...
   // Print out details from the Cloud Audit Logging entry
-  const payload = cloudevent.data.protoPayload;
-  console.log('Principal:', payload.authenticationInfo.principalEmail);
+  const metadata =
+    cloudevent.data &&
+    cloudevent.data.protoPayload &&
+    cloudevent.data.protoPayload.requestMetadata;
+  if (metadata) {
+    console.log('Caller IP:', metadata.callerIp);
+    console.log('User Agent:', metadata.callerSuppliedUserAgent);
```
Comment (averikitsch) [inline functions/v2/index.js:78]: nit: these might not be interesting either. Do we have an interesting use case? What is usually in the protoPayload?

File: functions/v2/test/index.test.js [hunk 2]
```diff
@@ -18,6 +18,9 @@ const {exec} = require('child_process');
 const waitPort = require('wait-port');
 
 const startFF = async (target, signature, port) => {
+  console.log(
```
Comment (averikitsch) [inline functions/v2/test/index.test.js:21]: nit: console.log

**Review by averikitsch — APPROVED** [review]
just nits

File: functions/v2/test/index.test.js (hunk 2 above)
Comment (ace-n) [reply functions/v2/test/index.test.js:21]: Good catch, thanks!

File: functions/v2/index.js (hunk 1 above)
Comment (ace-n) [reply functions/v2/index.js]: Resolved over chat - we decided to show some other properties, too.

**Review by JustinBeckwith — APPROVED** [review]

**Pull request merged**